spk/owfs/Makefile

```diff
@@ -0,0 +1,44 @@
+SPK_NAME = owfs
+SPK_VERS = 3.1p0
+SPK_REV = 1
+SPK_ICON = src/owfs.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = OWFS is an easy way to use the powerful 1-wire system of Dallas
+STARTABLE = yes
+DISPLAY_NAME = Owfs
+CHANGELOG = "1. Initial release, just owserver and owhttp"
+
+ADMIN_PORT = 2121
+RELOAD_UI = yes
+
+HOMEPAGE = http://owfs.org
+LICENSE  = GPL-2.0+
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+ADDITIONAL_SCRIPTS =
+
+FWPORTS          = src/${SPK_NAME}.sc
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+WIZARDS_DIR      = src/wizard/
```

Comment:
Could you please move this up, so you end up with INSTALLER, SSS, FWPORTS, WIZARDS without extra line breaks. Consistent formatting between packages makes things easier to see.